src/stacks-hierarchy/Root_TicketAssistantStack/types.ts

```diff
@@ -0,0 +1,41 @@
+import {FareProductTypeConfig} from '@atb/configuration';
+import {PreassignedFareProduct} from '@atb/reference-data/types';
+import {TariffZoneWithMetadata} from '@atb/stacks-hierarchy/Root_PurchaseTariffZonesSearchByMapScreen';
+import {UserProfileWithCount} from '@atb/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Travellers/use-user-count-state';
+
+export type Traveller = {
+  id: string;
+  user_type: string;
+};
+export type TicketAssistantData = {
+  frequency?: number;
+  duration?: number;
+  traveller?: Traveller;
+  zones?: string[];
+  preassigned_fare_products: string[];
+};
+
+export type TicketResponseData = {
+  product_id: string;
+  fare_product: string;
+  duration: number;
+  quantity: number;
+  price: number;
+  traveller: {id: string; user_type: string};
+};
+
+export type RecommendedTicketResponse = {
+  total_cost: number;
+  tickets: TicketResponseData[];
+  zones: string[];
+  single_ticket_price: number;
+};
+
+export type PurchaseDetails = {
+  tariffZones: TariffZoneWithMetadata[];
+  userProfileWithCount: UserProfileWithCount[];
+  fareProductTypeConfig: FareProductTypeConfig;
+  preassignedFareProduct: PreassignedFareProduct;
+  ticket: TicketResponseData;
+  singleTicketPrice: number;
+};
```

Comment:
Maybe look into the name `PurchaseDetails`. When I hear it it seems more like details after an actual purchase, like a receipt.